spec/lib/models/locales_updater_configuration_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.